### ISSUE TITLE
backslash autocomplete issue

### ISSUE THREAD

**Issue by Be1zebub** [FPtje/Sublime-GLua-Highlight#53]
When im tryin to make numbered list like: "1. hi\n2. mom" autocomplete for some reason this package removes backslash (tested with disabled package, it works fine)
seems like its related to https://github.com/FPtje/Sublime-GLua-Highlight/blob/b3317025f7dfbd6e3134e76a1c35b98dba3331eb/Lua.tmLanguage#L135-L145

**Comment by Be1zebub** [comment]
no, i do not - its happens automatically for some reason.
ive make video with issue demonstration: https://youtu.be/5RfZI-ukFQo
representation code: https://pastebin.com/uCS4jUNU

**Comment by Be1zebub** [comment]
perhaps related issue:
when im tryin to type `"/" ..` slash removes when im type 1st dot.

**Comment by FPtje** [comment]
Hmm, I'm looking at the file, and it's a syntax highlighting file. This file gives meaning to the words in the code. Most of what it does is just assign colours to things. I don't understand how autocomplete would even look at this file, besides maybe trying to find some keywords from the regex matches.

I just tried it in Sublime Text 4, to see if I can reproduce it. I made sure I had _no_ extensions installed other than this one, and I cannot reproduce the behavior.

![backslash bug](https://user-images.githubusercontent.com/1202014/200132153-84b21062-469e-43c8-8866-29057992a370.gif)

What I think is happening is that there is another extension installed on your Sublime. That extension tries to be clever with syntax highlighting files to get autocompletions from it. This goes wrong with newlines. That's only a guess, though. I cannot reproduce it.
